### PR TITLE
added initial files

### PR DESCRIPTION
**Pull request by nimdolphin**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Alumni+Sans+Pinstripe&family=Bitter&display=swap" rel="stylesheet">    
+  <title>Cofico</title>
+</head>
+<body>
+  <div class="container">
+    <div class="main">
+      <div id="nav">
+        <div class="cofico">
```
Comment (kathrynowy) [inline index.html]: unnecessary div

File: index.html [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+            Sign in
+          </a>
+
+          <button class="button">Sign up</button>
+        </div>
+      </div>
+
+      <h1>The Strong Aroma of
+        <span class="colortext"> Coffee</span>
+          can Instantly Make You <span class="colortext">Feel Better</span>.</h1>
```
Comment (kathrynowy) [inline index.html]:
```suggestion
      <h1>
        The Strong Aroma of
        <span class="colortext"> Coffee</span>
        can Instantly Make You
        <span class="colortext">Feel Better</span>.
      </h1>
```

File: index.html [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+        <p>Regular black coffee (without milk or cream) is low in calories. In fact, a typical cup of black coffee
+          only contains around 2 calories. However, adding cream or sugar will increase the calorific value.</p>
+
+      <button class="button">Explore More</button>
+    </div>
+
+    <div class="cup-coffee">
```
Comment (kathrynowy) [inline index.html]: do you need this div?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+      <img src="./img/cup-of-coffee.png" alt="a cup of coffee">
+    </div>
+
+  </div>
+
+  <div class="container-2">
+    <div class="bags-coffee">
```
Comment (kathrynowy) [inline index.html]: do you need this div?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,369 @@
...
+        <img src="./icon/tea-leyer.svg" alt="tea leyer">
+        <h2>The Leyer</h2>
+        <p>Regular tea (without milk or cream) is low in calories. In fact, a typical cup of tea</p>
+        <button class="button-2">Explore More</button>
+      </div>
+
+      <div class="chocolate-coffee">
+        <img src="./icon/chocolate-coffee.svg" alt="casual chocolate coffee">
+        <h2>casual chocolate coffee</h2>
+        <p>Regular casual coffee (without milk or cream) is low in calories. In fact, a typical cup of coffee</p>
```
Comment (kathrynowy) [inline index.html:85]: move text after tag on a new line if it's long

File: index.html [hunk 6]
```diff
@@ -0,0 +1,369 @@
...
+
+      <div class="coffee-leyer">
+        <img src="./icon/coffee-leyer.svg" alt="coffee leyer">
+        <h2>Coffee Leyer</h2>
+        <p>Regular coffee (without milk or cream) is low in calories. In fact, a typical cup of coffee</p>
+        <button class="button-2">Explore More</button>
+      </div>
+  </div>
+
+    <h1 class="heiding">Buy Your Coffee</h1>
```
Comment (kathrynowy) [inline index.html]: heading

File: index.html [hunk 7]
```diff
@@ -0,0 +1,369 @@
...
+          <span id="price">$30.00</span>
+          <span id="not-price">$36.00</span>
+        </div>
+
+        <div class="review">
+          <a href="#">
+          <img src="./icon/rewiew.svg" alt="rewiew">
+        </a>
+          <a href="#">
+          3.2k Review
```
Comment (kathrynowy) [inline index.html]: дочерние элементы/текст - должны сопровождаться соответствующей разметкой

File: index.html [hunk 8]
```diff
@@ -0,0 +1,369 @@
...
+
+    <div class="choice">
+      <button class="left">
+        <img src="./icon/left.svg" alt="left button">
+      </button>
+      <img id="points" src="./icon/points.svg" alt="points">
+      <button class="right">
+        <img src="./icon/right.svg" alt="right button">
+      </button>
+    </div>
```
Comment (kathrynowy) [inline index.html]:
```suggestion
      <button class="left">
        <img src="./icon/left.svg" alt="left button">
      </button>
    
      <img id="points" src="./icon/points.svg" alt="points">
      
      <button class="right">
        <img src="./icon/right.svg" alt="right button">
      </button>
    </div>
```

File: index.html [hunk 9]
```diff
@@ -0,0 +1,369 @@
...
+
+        <button class="button">Shop Now</button>
+      </div>
+    </div>
+
+    <div class="choice">
+      <button class="left">
+        <img src="./icon/left.svg" alt="left button">
+      </button>
+      <img id="points" src="./icon/points.svg" alt="points">
```
Comment (kathrynowy) [inline index.html]:
```suggestion
      <img id="points" src="./icon/points.svg" alt="points" />
```

File: index.html [hunk 10]
```diff
@@ -0,0 +1,369 @@
...
+    <div class="container-5">
+      <div>
+        <h1 id="process">We Love Great Process</h1>
+        <div class="one">
+          <div class="number">
+            <div id="ol">1</div>
+            <div id="dashed"></div>
+          </div>
+        
+          <div class="texxx">
```
Comment (kathrynowy) [inline index.html]: what is texxx?

File: index.html [hunk 11]
```diff
@@ -0,0 +1,369 @@
...
+            <h2>Order Your Menu</h2>
+            <p>Regular coffee (without milk or cream) is low in calories. In fact, a typical cup of coffee</p>
+          </div>
+
+        </div>
+
+        <div class="one">
+          <div class="number">
+            <div id="ol">2</div>
```
Comment (kathrynowy) [inline index.html]: ol? name classes properly

File: index.html [hunk 12]
```diff
@@ -0,0 +1,369 @@
...
+
+        </div>
+
+      </div>
+
+        <div class="picture">
+          <img id="cappuccino" src="./img/cappuccino.png" alt="cappuccino">
+          <img id="three-cups" src="./img/three-cups.png" alt="cups of coffee">
+          <img id="coffee-milk" src="./img/coffee-with-milk.png" alt="coffee with milk">
+
```
Comment (kathrynowy) [inline index.html]: ?

File: styles.css [hunk 13]
```diff
@@ -0,0 +1,865 @@
...
+  color: var(--metallic-bronze);
+  max-width: 670px;
+  height: 84px;
+  margin-top: 30px;
+  padding-top: 20px;
+  padding-bottom: 30px;
+}
+
+.cup-coffee img {
+  width: 544.75px;
```
Comment (kathrynowy) [inline styles.css]: we don't use float numbers.

File: styles.css [hunk 14]
```diff
@@ -0,0 +1,865 @@
...
+  height: 609px;
+  padding-bottom: 100px;
+}
+
+.container-2 {
+  display: flex;
+  background-image: url(./img/light-background.png);
```
Comment (kathrynowy) [inline styles.css]: wrong syntax

File: styles.css [hunk 15]
```diff
@@ -0,0 +1,865 @@
...
+  padding-top: 30px;
+  margin-bottom: 30px;
+}
+
+.container-3 {
+  display: flex;
+}
+
+.tea {
+  background-image: url(./img/tea-background.png);
```
Comment (kathrynowy) [inline styles.css]: same

File: styles.css [hunk 16]
```diff
@@ -0,0 +1,865 @@
...
+  background-repeat: no-repeat;
+  border-radius: 50px 50px 300px 300px;
+  background-position: center;
+  margin: 10px;
+  padding: 40px;
+}
+
+.tea img {
+  width: 80px;
+  height: 73.62px;
```
Comment (kathrynowy) [inline styles.css]: .

File: styles.css [hunk 17]
```diff
@@ -0,0 +1,865 @@
...
+
+#cappuccino {
+  width: 270px;
+  height: 278.8px;
+  padding: 50px 20px 20px 120px;
+}
+
+#three-cups {
+  width: 270px;
+  height: 280.88px;
```
Comment (kathrynowy) [inline styles.css]: .

File: styles.css [hunk 18]
```diff
@@ -0,0 +1,865 @@
...
+  position: absolute;
+  top: 200px;
+  left: 430px;
+}
+
+#coffee-milk {
+  width: 270px;
+  height: 280.88px;
```
Comment (kathrynowy) [inline styles.css]: .

File: styles.css [hunk 19]
```diff
@@ -0,0 +1,865 @@
...
+  margin-bottom: 10px;
+}
+
+.contact {
+  padding-left: 80px;
+}
+
+.contact h2 {
+  width: 100px;
+  font-family: var(--font-family);
```
Comment (kathrynowy) [inline styles.css]: you can add to a body font family and remove it everywhere

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: index.html (hunk 5 above)
Comment (nimdolphin) [reply index.html:85]: I think that's ok

File: index.html (hunk 1 above)
Comment (nimdolphin) [reply index.html]: fixed

File: index.html (hunk 2 above)
Comment (nimdolphin) [reply index.html]: fixed

File: index.html (hunk 3 above)
Comment (nimdolphin) [reply index.html]: fixed

File: index.html (hunk 4 above)
Comment (nimdolphin) [reply index.html]: fixed

File: styles.css (hunk 15 above)
Comment (nimdolphin) [reply styles.css]: fixed

File: styles.css (hunk 16 above)
Comment (nimdolphin) [reply styles.css]: fixed

File: styles.css (hunk 17 above)
Comment (nimdolphin) [reply styles.css]: fixed

File: styles.css (hunk 18 above)
Comment (nimdolphin) [reply styles.css]: fixed

File: styles.css (hunk 19 above)
Comment (nimdolphin) [reply styles.css]: fixed

**Pull request merged**